pkg/pipeline/pipeline.go

```diff
@@ -128,7 +128,7 @@ func (p *Pipeline) Components(filter []string) ([]component.Component, error) {
 
 	components := make([]component.Component, 0)
 	for _, ns := range modules {
-		members, err := p.cm.Components(ns)
+		members, err := p.cm.Components(p.app, ns.Name())
```

Comment:
`module` used to be named `namespace`. That was changed for reasons, but there are still a bunch of `ns` littered through the code base. Would you mind cleaning up the ones in this PR?

Reply:
Sure thing!

Reply:
Pushed.